### PR TITLE
Finished problem 43.

### PR DESCRIPTION
**Pull request by ikhat**
Making this one it's own PR, since I suspect you might want me to cut down the solution to the first part a little bit. The solution here has quite a bit of detail about how to find the matrix of a transformation. All the steps needed to understand this are present in previous problems though, which is why I'm not sure if you want it all explained.

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -3084,10 +3084,57 @@ \section*{Task 2.3: Pat and Jamie}
 	and let $\mathcal R$ be rotation counter-clockwise by $90^\circ$.
 	\begin{parts}
 		\item Find a matrix $P$ so that $P\vec x=\mathcal P(\vec x)$ for all $\vec x\in\R^2$.
+			\begin{solution}
+				$P = \frac{1}{13}\mat{4&6\\6&9}$ is such a matrix.
+
+				We know from an earlier exercise that 
+				$\mathcal P(\vec x)=\dfrac{\vec x\cdot\vec u}{\norm{u}^2}\vec u$.
+				So if $\Set{\vec e_1,\vec e_2}$ is the standard basis for $\R^2$,
+				we can compute
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: I'd like you to do these by plugging in e_1 and e_2 (which are `\xhat` and `\yhat` in this document) and then pointing out that these give the first and second columns

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -3084,10 +3084,57 @@ \section*{Task 2.3: Pat and Jamie}
...
-			for $\mathcal P\circ\mathcal R$ and $\mathcal R\circ \mathcal P$.
+			\begin{solution}
+				$R=\mat{0&-1\\1&0}$ is such a matrix, 
+				found using the same reasoning as the previous part
+			\end{solution}
+		\item Write down matrices $A$ and $B$ for $\mathcal P\circ\mathcal R$ 
+			and $\mathcal R\circ \mathcal P$.
+			\begin{solution}
+				$A=\frac{1}{13}\mat{6&-4\\9&-6}$ and $B=\frac{1}{13}\mat{-6&-9\\4&6}$
+				are two such matrices, found using the same reasoning as above.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Do this out with e_1 and e_2 (and not with matrix mult).

File: problemsets-mat223/linearalbegra.tex (hunk 1 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: I can do it that way.

I see that you define `\xhat` and `\yhat` in the preamble, but they never appear in the rest of the document, and you call the standard basis vectors \vec e_1 and \vec e_2 every time.

Is this something you want me to change everywhere?

Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: Yes, please. Unless we talk about an arbitrary e_i, we should use \xhat, etc. This way someone could recompile the document for their physics class :-)

Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Fair enough. I agree that it would be nicer that way. 

Is it okay if I save the changing e_1,e_2 into \xhat,\yhat throughout the whole document to another commit after I've finished all the problems? I can change them as I come across them, but I'd rather wait until I'm done the later problems before going back to the beginning and changing the stuff I've gone over already.

File: problemsets-mat223/linearalbegra.tex (hunk 2 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: When I wrote "the same reasoning as above", that's what I meant. I figured it didn't need to be written out again. I can do that though.

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -3084,10 +3084,87 @@ \section*{Task 2.3: Pat and Jamie}
...
+				We know from an earlier exercise that 
+				$\mathcal P(\vec x)=\dfrac{\vec x\cdot\vec u}{\vec u\cdot\vec u}\vec u$.
+				So if $\Set{\xhat,\yhat}$ is the standard basis for $\R^2$,
+				we can compute
+				\[
+					\mathcal P(\xhat)=\frac{2}{13}\vec u=\frac{1}{13}\mat{4\\6}
+					\qquad \text{and} \qquad
+					\mathcal P(\yhat)=\frac{3}{13}\vec u=\frac{1}{13}\mat{6\\9}.
+				\]
+				So since we know $\mathcal P$ is linear, for any $\vec x=\mat{x\\y}$,
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: I'd like you to rephrase this as:

Suppose P is the matrix corresponding to \mathcal P. For any matrix, multiplying by $\xhat$ gives
the first column and multiplying by $\yhat$ gives the second. Therefore the first column of P is P\xhat=... and
the second column is P\yhat=...

Similarly for the others too, please. I'd like to emphasize the standard reasoning which is "when you don't know, make a matrix of unknowns and find a system of equations", which gives students something to grasp onto when they're lost.
